code/defines/obj/weapon.dm

```diff
@@ -412,7 +412,7 @@
 	var/breakouttime = 60
 	var/armed = 0
 	var/trapped = 0
-	var/datum/organ/external/trappedorgan = "LIMB_LEFT_LEG"
+	var/datum/organ/external/trappedorgan //The limb currently trapped, it must be a leg
```

Comment:
iirc it can be an arm if you try to touch with hand

Reply:
Checked attack_hand, it can't, it only damages your arm and then deactivates itself